docs/serviceprincipal.md

```diff
@@ -25,12 +25,31 @@ There are several ways to create a Service Principal in Azure Active Directory:
    az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/${SUBSCRIPTION_ID}"
    ```
 
-This will output your `client_id`, `client_secret` (password), `sp_name`, and `tenant`.  The `sp_name` or `client_id` may be used for the `servicePrincipalProfile.servicePrincipalClientId` and the `client_secret` is used for `servicePrincipalProfile.servicePrincipalClientSecret`.
+You can set `SUBSCRIPTION_ID` like this:
 
-Confirm your service principal by opening a new shell and run the following commands substituting in `sp_name`, `client_secret`, and `tenant`:
+```
+SUBSCRIPTION_ID=$(az account list --query "[?name == 'subscription name'].[id]" --out tsv)

```

Comment:
This is good for people know there subscription name.
Can you also add more comment for people set their SUBSCRIPTION_ID directly by the id ?